telco-hub/configuration/reference-crs/required/storage/odf-internal/odfNS.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
ODF is optional if they have an external object store for observability

Reply:
should we move ODF and LSO under optional ?

Reply:
good